src/jumio-kyc/kyc.controller.spec.ts

```diff
@@ -227,7 +227,12 @@ describe('KycController', () => {
         .expect(HttpStatus.OK);
 
       expect(body).toMatchObject(
```

Comment:
If the user has a kyc that is in progress, then when the `GET` endpoint is hit `attemptable` should be false, no?